src/main/java/ChatBot.java

```diff
@@ -0,0 +1,117 @@
+//import java.util.Scanner;
+//
+//public class ChatBot {
+//    private static final Scanner in = new Scanner(System.in);
+//
+//    private static final String DIVIDER = "____________________________________________________________";
+//    private static final int TASK_SIZE = 100;
+//    private static final String MESSAGE_WELCOME = "How can I assist you? Type something below! :D";
+//
+//    /**
+//     * Prints a divider for ChatBot
+//     */
+//    private void printDivider() {
+//        System.out.println(DIVIDER);
+//    }
+//
+//    private void startChat() {
+//
+//    }
+//
+//    /**
+//     * Begins the task state until exit is called
+//     */
+//    public void startToDo() {
+//        printDivider();
+//        System.out.println("You have entered todo mode! Type exit to quit this mode.");
+//        System.out.println("Current commands: list, done (number)");
```

Comment:
Shouldn't this be extracted out into a MACRO. You can put it as a MACRO as MESSAGE_TODO_MODE